### ISSUE TITLE
When updating the portable version of NVDA, a warning dialog box pops up

### ISSUE THREAD

**Issue by wmhn1872265132** [nvaccess/nvda#16847]
### Steps to reproduce:

1. Use an old version of NVDA (such as alpha-32530,e57d7e54 (2024.3.0.32530)) to create a portable version
2. Start the NVDA
3. Check for and install updates

### Actual behavior:

During the installation process, the following two dialog boxes will appear.

1. > Portable Copy Exists dialog A portable copy already exists in the directory 'F:\Program Files\NVDA'. Do you want to update it?
2. > Success dialog Successfully created a portable copy of NVDA at F:\Program Files\NVDA

### Expected behavior:

The above dialog box should not appear and updating the portable version of NVDA should have the same behavior as the installed version

#### NVDA installed/portable/running from source:
portable

#### NVDA version:
alpha-32865,e2541eef(2024.4.0.32865)

#### Windows version:
Windows 10 21H2 (AMD64) build 19044.4651

#### Name and version of other software in use when reproducing the issue:
none

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Dialog 1: All versions after https://github.com/nvaccess/nvda/pull/16686 merge;
Dialog 2: All versions over the years

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by wmhn1872265132** [comment]
### Additional notes on https://github.com/nvaccess/nvda/pull/16686
1. The issue number cited in changes is incorrect
> By default, a new folder will be created when making a portable copy. A warning message will inform you if you try writing to a non-empty directory. (#16684)
2. Although the following string can be translated, it still displays English in the target language
- > A portable copy already exists in the directory '{portableDirectory}'. Do you want to update it?
- > The specified directory '{portableDirectory}' is not empty. Proceeding will delete and replace existing files in the directory. Do you want to overwrite the contents of this folder?

**Comment by Adriani90** [comment]
cc: @seanbudd I think this is a regression introduced in #16686.
